sbws/util/state.py

```diff
@@ -0,0 +1,62 @@
+from sbws.util.filelock import FileLock
+import os
+import json
+
+
+class State:
+    _ALLOWED_TYPES = (int, float, str, bool, type(None))
```

Comment:
i'd add also here a docstring or comment about what is this class for and which format it's supposed to store/read (json)